### PR TITLE
[*] Technical - Clean some resources services

### PR DESCRIPTION
**Pull request by VincentMolinie**
## Pull Request checklist:

- [ ] Write an explicit title for the Pull Request
- [ ] Write changes made in the CHANGELOG.md
- [ ] Create automatic tests
- [ ] Test manually the implemented changes
- [ ] Review my own code (indentation, syntax, style, simplicity, readability)
- [ ] Wonder if you can improve the existing code

### CODE REVIEW

File: src/services/resources-exporter.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            }
+            return null;
+          });
+      });
+  }
+
+  this.perform = dataSender => retrieveBatch(dataSender, BATCH_INITIAL_PAGE);
+}
+
+module.exports = ResourcesExporter;
```
Comment (VincentMolinie) [inline src/services/resources-exporter.js]: This file has just been moved

Comment (arnaudbesnier) [reply src/services/resources-exporter.js]: ...and syntax changed.

**Comment by VincentMolinie** [comment]
[(13) Smart Routes - Expose liana methods](https://trello.com/c/c7xhPwak/3869-13-smart-routes-expose-liana-methods)

File: src/services/resources-exporter.js (hunk 1 above)
Comment (VincentMolinie) [reply src/services/resources-exporter.js]: Auto fix of eslint

**Pull request merged**